### ISSUE TITLE
Add more logging to the pytorch lighning models.

### ISSUE THREAD

**Issue by fstroth** [airctic/icevision#600]
The feature consists of two parts:
  1. Add the validation loss to the progress bar by default
  2. Create boolean parameter for extended progress bar logging (showing the different components of the loss)
